tests/integration/targets/fusion_hap/tasks/main.yml

```diff
@@ -18,7 +18,7 @@
   environment: "{{ test_env }}"
 - name: Validate the task
   ansible.builtin.assert:
-    that: "'hap_foo' in fusion_info['fusion_info']['hosts']"
+    that: "'hap_foo' in fusion_info['fusion_info']['host_access_policies']"
```

Comment:
Potential enhancement: Since you've added regions dict, these types of checks can now be also added to [fusion_region/tasks/main.yml](https://github.com/Pure-Storage-Ansible/Fusion-Collection/blob/1995ec73125863474ecc4a8f3a0bc4fe0110e3e4/tests/integration/targets/fusion_region/tasks/main.yml)

Reply:
Good idea :)